rag/llm/rerank_model.py

```diff
@@ -154,8 +156,8 @@ def similarity(self, query: str, texts: list):
 
 class XInferenceRerank(Base):
     def __init__(self, key="xxxxxxx", model_name="", base_url=""):
-        if base_url.split("/")[-1] != "v1":
```

Comment:
I recommand the code changed as follows:
`if base_url.split("/")[-1] != "v1" and base_url.split("/")[-1] != "rerank":`

Reply:
In this case, the end is / easy to have problems, so I used find